python/test/unit/runtime/test_cache.py

```diff
@@ -522,7 +509,6 @@ def cache_hook(*args, **kwargs):
     assert specialization_data is not None
 
     # clear the cache
-    reset_tmp_dir()
```

Comment:
it seems like we are not testing clearing the disk cache anymore?

Reply:
oh, right, I see now that the `warmup()` call was populating the cache. Let me add an `rmtree` call here...